game/src/level/elements/ILevel.java

```diff
@@ -21,20 +21,17 @@ default void setRandomStart() {
 
     /** Mark a random tile as end */
     default void setRandomEnd() {
-        Tile newEnd = getRandomTile(LevelElement.FLOOR);
-        while (newEnd == getStartTile()) {
-            newEnd = getRandomTile(LevelElement.FLOOR);
+        List<FloorTile> floorTiles = getFloorTiles();
+        if (floorTiles.size() <= 1) {
+            // not enough Tiles for startTile and ExitTile
```

Comment:
klingt für mich so, als müsste man hier eine Exception werfen, oder? 
Ohne Ausgang funktioniert das Spiel ja nicht

Reply:
Ich hatte es jetzt erstmal drinnen gelassen grund dafür wir haben später ja bei den Räumen ja nicht mehr in jedem Level einen Ausgang. Und hier ist einfach nur sichergestellt, wenn es nicht möglich ist, dann gibt es erstmal keinen, wenn du aber lieber eine Exception haben willst, kann ich dir gerne eine reinbauen.

Reply:
Stimmt... an die Räume hab ich garnicht gedacht. 
Dann lass so